src/scripts/main.js

```diff
@@ -1 +1,19 @@
 'use strict';
+
+const ul = document.querySelector('#thumbs');
+const largeImg = document.querySelector('#largeImg');
+
+ul.addEventListener('click', function(e) {
+  e.preventDefault();
+
+  const isClosest = e.target.closest('.list-item');
+  const itContains = ul.contains(e.target);
```

Comment:
```suggestion
  const closestLink = e.target.closest('.list-item__link');
  const isInsideUl = ul.contains(e.target);
```
Находи сразу ссылку, из которой будешь брать href, а не li, в которой лежит ссылка